src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/services_instanceof_ignore_class.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <instanceof id="Symfony\Component\DependencyInjection\Tests\Fixtures\BarInterface" lazy="true" autowire="true">
+            <tag name="foo" />
+            <ignore name="Symfony\Component\DependencyInjection\Tests\Fixtures\Baz" />
```

Comment:
`<ignore>Symfony\Component\DependencyInjection\Tests\Fixtures\Baz</ignore>`?

Reply:
I've followed the same signature of tags but I agree that your version is better.

Reply:
tags allow having arbitrary attributes